wp-content/themes/pub/wporg-learn-2024/src/style/style.scss

```diff
@@ -39,3 +39,28 @@ body {
 		padding-top: unset !important;
 	}
 }
+
+.wporg-learn-sidebar-meta-info {
+	.wp-block-sensei-lms-button-take-course {
+		margin-top: 0;
+		margin-bottom: var(--wp--preset--spacing--20);
+
+		> button {
+			line-height: 1;
+		}
+	}
+
+	.wporg-learn-sidebar-playground  a {
+		line-height: 1;
+	}
+
+	.wporg-learn-sidebar-all-courses {
+		a {
+			text-decoration: none;
+		}
+
+		+ * {
+			margin-top: 16px;
```

Comment:
Could you apply this to the bottom of .wporg-learn-sidebar-all-courses with block markup instead?

Reply:
I added here because `sensei-lms/course-progress` block has [default margin-top](https://github.com/Automattic/sensei/blob/trunk/assets/blocks/shared-style.scss#L6-L9), if move this to the block markup, it still needs setting margin-top: 0 to overwrite the default value. And [it](https://github.com/Automattic/sensei/blob/trunk/assets/blocks/course-progress-block/block.json) doesn't seem to provide a controllable margin.

Reply:
It's moved to course-progress selector. 0b96d32